src/Cart.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { deleteOrderProductsThunk, updateOrderProductThunk, se } from './store';
+import { setOrderProductsThunk, setOrdersThunk, setProductsThunk, setUsersThunk} from './store'
+
+
+
+class _Cart extends React.Component {
+  constructor(props) {
+    super();
+    this.deleteItem = this.deleteItem.bind(this);
+    this.updateItem = this.updateItem.bind(this);
+  }
+  // async componentDidMount() {
+  //   await this.props.getUsers()
+  //   await this.props.getProducts()
+  //   await this.props.getOrders()
+  //   await this.props.getOrderProdcuts()
+  // }
+  async deleteItem (id){
+    await this.props.deleteItem(id)
+  }
+  async updateItem (item) {
+    await this.props.updateItem(item)
+  }
+  render(){
+    const { orders , products, orderProducts, auth, match } = this.props;
+    // console.log('orders', orders)
+    const cart = orders.find(order => order.userId === match.params.id && order.status ==='cart');
```

Comment:
Lets build a backend route that we can give a userId that returns our 'cart' to us. This is by no means broken or even necessarily 'bad', but I think it makes for cleaner React code to not have to do this.